### ISSUE TITLE
luascript -> Game:createMonster and onSpawn don't register creaturescripts

### ISSUE THREAD

**Issue by ArturKnopik** [opentibiabr/otservbr-global#1190]
When we create a monster with Game.createMonster("monsterName", position) the monster doesn't register events with "onSpawn"
```
function Monster:onSpawn(position)
	self:registerEvent("someRandomCreaturescripts") --reduction of damage received
end
```
my way to solve this problem is:
in 
`#include "events.h"`
and
`extern Events* g_events;`
and in int `LuaScriptInterface::luaGameCreateMonster(lua_State* L)`
after `Creature* master = getCreature(L, 5);`
add this `g_events->eventMonsterOnSpawn(monster, position);`

**Comment by lBaah** [comment]
Use `local monster = Game.createMonster...`
and then `monster:registerEvent...`

So you don't set that event checking for every monster you create.
This is not a bug since it is not a SPAWN.

**Comment by ElTeixe** [comment]
Agree, is not a bug due the monster isnt created by the Spawn. This probably can be closed.

**Comment by Beats-Dh** [comment]
This probably can be closed.

**Comment by ArturKnopik** [comment]
let's think about it, if we have an event in onSpawn then every monster created with game.createMonster we need manually  add an event from onSpawn, usually if we have something in onSpawn we want to have it in EVERY monster.
For me, adding this will make it easier to create custom OTBR-based servers

This is my point of view

**Comment by Beats-Dh** [comment]
> let's think about it, if we have an event in onSpawn then every monster created with game.createMonster we need manually add an event from onSpawn, usually if we have something in onSpawn we want to have it in EVERY monster.
> For me, adding this will make it easier to create custom OTBR-based servers
> 
> This is my point of view

Seeing your point of view, yes it is valid. I'm not sure how to do it, if I can, I could open a PR.

**Comment by ArturKnopik** [comment]
https://github.com/ArturKnopik/otservbr-global/tree/game_createMonsterSpawnCreaturescripts look on this, small changes, may affect server startup time
